engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-traces.html

```diff
@@ -16,6 +16,18 @@
     </p>
   </div>
 
+<p bst-alert="info" ng-hide="host.content_facet_attributes.katello_tracer_installed">
```

Comment:
As much as I dislike the non-semanticness of `div` elements, I feel like this and the one below it should be `<div>`s instead of `<p>`s, just to remain consistent with all the others in Katello.